frontend/src/pages/TrainingReportForm/index.js

```diff
@@ -156,7 +156,8 @@ export default function TrainingReportForm({ match }) {
       }
 
       try {
-        const users = await getTrainingReportUsers(eventRegion);
+        const eventId = hookForm.getValues('id');
```

Comment:
Putting "hookForm" inside the useEffect causes this data to be fetched 6 times, I think we might want to refactor that

Reply:
I switched to use trainingReportId this comes in from the route match. I only see two useEffect calls now.